VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -30,8 +30,8 @@ public void setTracker(ISegmentTracker tracker) {
         this.tracker = tracker;
     }
 
-    class VideoEvent {
-        public VideoEvent() {
+    class SegmentAnalyticsEvent {
+        public SegmentAnalyticsEvent() {
```

Comment:
Nitpick: Since this class is now generalized to accommodate non-video events, any methods that were specific to videos/courses should be renamed.  For example, `setContext` should probably be renamed to `setCourseContext`.  As it currently is implemented, `setContext` does not apply to events outside the context of a course.